scripts/deploy-federation.sh

```diff
@@ -49,44 +49,6 @@ set -o pipefail
 
 source "$(dirname "${BASH_SOURCE}")/util.sh"
 
-function deploy-with-script() {
-  # Create a permissive clusterrolebinding to allow federation controllers to run.
-  if [[ "${NAMESPACED}" ]]; then
-    # TODO(marun) Investigate why cluster-admin is required to view cluster registry clusters in a namespace
-    kubectl -n "${NS}" create rolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
-  else
-    # TODO(marun) Make this more restrictive.
-    kubectl create clusterrolebinding federation-admin --clusterrole=cluster-admin --serviceaccount="${NS}:default"
```

Comment:
because we will not create `clusterrolebinding` `federation-admin`, we should clean code to avoid useless deletion of this resource in `scripts/delete-federation.sh`